src/programs/ps-lut/main.cpp

```diff
@@ -1,40 +1,42 @@
-#include <iostream>
-#include <vector>
-#include <string>
+#include <boost/format.hpp>
 #include <boost/program_options.hpp>
 #include <boost/shared_ptr.hpp>
-#include <boost/format.hpp>
-#include <pokerstove/util/combinations.h>
+#include <iostream>
 #include <pokerstove/peval/Card.h>
 #include <pokerstove/peval/CardSet.h>
 #include <pokerstove/peval/CardSetGenerators.h>
 #include <pokerstove/peval/PokerHandEvaluator.h>
+#include <pokerstove/util/combinations.h>
+#include <string>
+#include <vector>
 
 using namespace std;
 namespace po = boost::program_options;
 using namespace pokerstove;
 
-int main (int argc, char ** argv)
+int main(int argc, char** argv)
 {
-    try 
+    try
     {
-        // set up the program options, handle the help case, and extract the values
-        po::options_description desc(
-            "create a lookup table for poker hands\n");
-        desc.add_options()
-            ("help,?",    "produce help message")
```

Comment:
This is the preferred format. Structured for readability.

Reply:
This is reformatted manually but not showing up on this diff.